apps/payments/next/app/[locale]/en.ftl

```diff
@@ -33,6 +33,7 @@ intent-payment-error-get-in-touch = Hmm. There was a problem authorizing your pa
 intent-payment-error-generic = An unexpected error has occurred while processing your payment, please try again.
 intent-payment-error-insufficient-funds = It looks like your card has insufficient funds. Try another card.
 general-paypal-error = An unexpected error has occurred while processing your payment, please try again.
+paypal-active-subscription-no-billing-agreement-error = It looks like there was a problem billing your { -brand-paypal } account. Please re-add a billing agreement.
```

Comment:
Thanks for adding the term.

Re-reading this string, I'm not sure what a "billing agreement" is or what I would need to do to add one.This will definitely be hard to make consistent/understandable for localization. Is it possible to rephrase this differently like "re-add automatic payment" or something like that?